steps/datadog-mobile-app-run-tests/1.0.0/step.yml

```
@@ -0,0 +1,113 @@
+title: datadog-mobile-app-run-tests
+summary: |
+  Step to run Datadog Synthetics tests on your CI
+description: |
+  With the `synthetics-test-automation-bitrise-step-run-tests` step you can run Synthetics tests during your Bitrise CI and ensure all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle.
+website: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests
+source_code_url: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests
+support_url: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/issues
+published_at: 2024-03-01T10:58:14.103768+01:00
+source:
+  git: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git
+  commit: 2239150885d7cce9cec0abdd4072cd51b01ecdc0
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: run-tests.sh
+inputs:
+- api_key: $DATADOG_API_KEY
+  opts:
+    description: The API key used to query the Datadog API.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: Datadog API Key
+- app_key: $DATADOG_APP_KEY
+  opts:
+    description: The application key used to query the Datadog API.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: Datadog APP Key
+- config_path: ""
+  opts:
+    description: The global JSON configuration used when launching tests.
+    title: Global JSON configuration
+- device_ids: ""
+  opts:
+    description: Override the mobile device(s) to run your mobile test.
+    title: Device IDs
+- fail_on_critical_errors: false
+  opts:
+    description: A boolean flag that fails the CI job if no tests were triggered,
+      or results could not be fetched from Datadog.
+    title: Fail on critical errors
+- fail_on_missing_tests: false
+  opts:
+    description: Fail if at least one specified test with a public ID (using `public_ids`
+      or listed in a test file) is missing in a run (for example, if it has been deleted
+      programmatically or on the Datadog site).
+    title: Fail on missing tests
+- fail_on_timeout: true
+  opts:
+    description: A boolean flag that fails the CI job if at least one test exceeds
```

Is there a way you could add a list of possible values or an example of how the input should look like for everywhere it is applicable? description/summary can be multiple lines if needed.

```suggestion
    description: A boolean flag that fails the CI job if at least one test exceeds. |-
      Possible values:
      - `true`
      - `false`
```
See an example: https://github.com/bitrise-steplib/steps-xcode-test/blob/4e14f19dd7ad5a9bcb1318bcbdab3b8d9ef63ce3/step.yml#L98

---

We have other integrations that we want to keep aligned to as close as possible. I did add however a link with examples for all inputs in the description 👍